<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>451</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QRadioButton" name="rbMale">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>98</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>男</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbFemale">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>98</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>女</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>451</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QRadioButton" name="rbCN">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>98</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>中国</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbUS">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>98</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>美国</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textOut">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>431</width>
     <height>64</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
